doc/version-history.rst

```diff
@@ -0,0 +1,17 @@
+.. _Version_History:
+
+===============
+Version History
+===============
+
+.. At the time of writing the Version history/release notes are not yet standardized amongst CSCs.
+.. Until then, it is not expected that both a version history and a release_notes be maintained.
+.. It is expected that each CSC link to whatever method of tracking is being used for that CSC until standardization occurs.
+.. No new work should be required in order to complete this section.
+.. Below is an example of a version history format.
+
+v0.4.1
```

Comment:
Remove this?

Reply:
Thats just the template showing an example of the version history format.